packages/guui/index.js

```diff
@@ -2,11 +2,12 @@
 
 // provide a consistent wrapper around the libs guui depends on.
 
-import { Component, render as hydrateApp } from 'preact';
-import preactRenderToString from 'preact-render-to-string';
 import { hydrate as hydrateCSS } from 'emotion';
-import styled from 'preact-emotion';
 import { extractCritical } from 'emotion-server';
+import { Component } from 'react';
+import { hydrate as hydrateApp } from 'react-dom';
+import { renderToString as reactRenderToString } from 'react-dom/server';
```

Comment:
Shall we just keep the default names?

Reply:
Yeah it was easier at the time of converting not to alter the interface `guui` exports, which I'd need to do if I used the default names as some clash with our own functions.